temp-legacy-code/src/main/java/com/ivy/legacy/datamodel/TagTransaction.kt

```diff
@@ -0,0 +1,16 @@
+package com.ivy.legacy.datamodel
+
+import com.ivy.data.db.entity.TagTransactionEntity
+import java.util.*
+
+
+data class TagTransaction(
+    val tagId: UUID,
+    val associatedId: UUID,
```

Comment:
Let's use typed-ids. Is this `TransactionId`? If so let's call it `associatedTranssctionId: TransactionId`